tools/tests/py2fgen/test_dycore_wrapper.py

```diff
@@ -256,6 +245,7 @@ def test_dycore_wrapper_granule_inputs(
 
     # PrepAdvection
     vn_traj = sp.vn_traj()
+    vol_flx_ic = data_alloc.zero_field(icon_grid, dims.CellDim, dims.KDim)  # TODO sp.vol_flx_ic()
```

Comment:
hm, you wanted the `vol_flx_ic` in the init savepoint? I think I only added it to the exit... shame on me...